### ISSUE TITLE
Proxmox Certificate format error

### ISSUE THREAD

**Issue by rubeste** [glatzert/ACME-Server-ADCS#43]
I have a homelab with a ADCS server running and I am trying this app. After some configuration I got it basically working. However when I use proxmox to submit an ACME request it fails with the message:

`TASK ERROR: POST of 'https://acme.local.example.com/order/XXXXXXXXXXXXXXXXX/certificate' failed - Server reply does not look like a PEM encoded certificate`
I have seen one issue with a similar message, but it seems not to be relevant. The server the ACME is running on is as follows:

- Windows Server 2022
- Using gMSE account: AcmeService
- IIS Sites
  - Acme Site:
    - App files: C:\acmeserver
    - ACME-ADCS directory: C:\ACME-ADCS

If you need more information you can all ways ask.

**Comment by glatzert** [comment]
Does Proxmox by any chance save the downloaded data, so we can take a look inside?

**Comment by rubeste** [comment]
I don't think it does and I can not make sense of the pearl scripts Proxmox uses: [pve-manager](https://github.com/proxmox/pve-manager/blob/master/PVE/API2/ACME.pm)

**Comment by madtempest** [comment]
Given certbot also seems to have a Proxmox specific installation plugin, it probably stands to reason to redirect to certbot for that specific use case.
